cmake/ecbuild_check_os.cmake

```diff
@@ -424,9 +424,15 @@ if( WIN32 )
 
   set( EC_OS_NAME "windows" )
 
+  find_program( BASH NAMES bash )
+
+  if( NOT BASH )
+      ecbuild_critical("Bash not found")
```

Comment:
This error message should be more helpful for the user. For example helping him to understand how can he provide the path to bash.
e.g. "Could not find program 'bash' - You may specify with -DBASH_EXE=/path/to/bash"

Reply:
Done